server/index.ts

```diff
@@ -21,12 +30,12 @@ const spotifyClientId = process.env.SPOTIFY_CLIENT_ID;
 const spotifyClientSecret = process.env.SPOTIFY_CLIENT_SECRET;
 const spotifyRedirectUri = process.env.SPOTIFY_REDIRECT_URI;
 
-// mostly for redirecting
+// redirect endpoint -- mostly a helper route
 app.get('/', (_req, res) => {
   res.send('redirecting...');
 });
 
-// login endpoint
+// login endpoint -- user enters login credentials and spotify authorizes via OAuth 2.0
 app.get('/auth/login', (_req, res) => {
```

Comment:
While naming your unused param(s) `_req` is not wrong by any means, this is a more common convention:

```suggestion
app.get('/auth/login', (_, res) => {
```